life_expectancy/tests/test_cleaning.py

```diff
@@ -1,15 +1,16 @@
 """Tests for the cleaning module"""
+import pytest
 import pandas as pd
 
-from life_expectancy.cleaning import clean_data
-from . import OUTPUT_DIR
 
+from life_expectancy.data_cleaning import clean_data
 
-def test_clean_data(pt_life_expectancy_expected):
+
+@pytest.mark.unit
```

Comment:
Não utilizei este decorator, isto é para forçar a recolher como uma unidade de tests?

Reply:
sim, assim consegues executar se quiseres testes de integração ou unitários em separado, e fica mais claro quais são quais.